<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-2NA4FFI8" failures="1" tests="1" name="Smoke_Test_inChrome" time="24.188" errors="0" timestamp="2022-07-28T08:38:14 IST">
  <testcase classname="MindQ.SeleniumTestngFrameworkJune.LoginTestCaseFromExcel" name="loginUsingInvalidCredentialsAndWriteResult" time="8.506">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Test case is failed expected [false] but found [true],
	Test case is failed expected [false] but found [true],
	Test case is failed expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Test case is failed expected [false] but found [true],
Test case is failed expected [false] but found [true],
Test case is failed expected [false] but found [true]
at MindQ.SeleniumTestngFrameworkJune.LoginTestCaseFromExcel.loginUsingInvalidCredentialsAndWriteResult(LoginTestCaseFromExcel.java:131)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginUsingInvalidCredentialsAndWriteResult -->
</testsuite> <!-- Smoke_Test_inChrome -->
